src/ref.builds

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="*\ref\**\*.*proj">
+    <Project Include="*\ref\**\*.builds;*\ref\**\*.*proj">
```

Comment:
I see I was reviewing individual commits. Looking at your final set of changes I like that the default is netcoreapp1.1 for these guys, and you just added the APIs straight up without #ifdefs. 

I do think we should probably seed the .builds files in the ref folders so we have consistency and the cross-compile will just work if and when we need it.